pkg/registry/core/shoot/strategy.go

```diff
@@ -170,6 +176,9 @@ func (shootStrategy) Validate(_ context.Context, obj runtime.Object) field.Error
 	shoot := obj.(*core.Shoot)
 	allErrs := field.ErrorList{}
 	allErrs = append(allErrs, validation.ValidateShoot(shoot)...)
+	if err := validation.ValidateForceDeletion(shoot, nil); err != nil {
+		allErrs = append(allErrs, err)
+	}
```

Comment:
Why call it here and not as part of `ValidateShoot`? Adding more and more validation-related stuff to the strategy files makes them more complicated IMO. We should always try to keep the validation in a single place and only add special code here for very special situations.

Reply:
If we move it to `ValidateShoot` for creation, then it will be called twice on Update.